packages/platform-ios/src/config/findPodspec.ts

```diff
@@ -1,11 +1,11 @@
 import glob from 'glob';
 import path from 'path';
 
-export default function findPodspec(folder: string): string | null {
+export default function findPodspec(folder: string): string | undefined {
   const podspecs = glob.sync('*.podspec', {cwd: folder});
 
   if (podspecs.length === 0) {
-    return null;
+    return;
```

Comment:
Just FYI: I don't see a reason for this change, but I don't have strong opinions here

Reply:
Originally, I did it because I wanted to use `poodfile?: string` and that was incompatible with null being returned here.

Not an issue now that we have `string | undefined` vs `string | null`, I think I can revert this change anyway.